lib/kitchen/directions/bake_learning_objectives.rb

```diff
@@ -9,9 +9,13 @@ def self.v1(chapter:)
         end
       end
 
-      def self.v2(chapter:, add_title: true)
+      def self.v2(chapter:, add_title: true, li_numbering: false)
         learning_objectives =
-          chapter.abstracts.any? ? chapter.abstracts : chapter.learning_objectives
+          if %i[in_appendix count_only_li_in_appendix].include?(li_numbering)
```

Comment:
i see what this is doing, but the structure is confusing to read. did rubocop try to stop you from saying `if li_numbering == :in_appendix or li_numbering == :count_only_li_in_appendix`? if not, i think that syntax is slightly easier to read

Reply:
Yes it did. :smile:  I have used the recommended solution.